tests/wasm/host_binding.rs

```diff
@@ -0,0 +1,40 @@
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_test::*;
```

Comment:
I guess we don't really have any compile fail tests, but it would be nice to test what happens when both `host_binding` and `structural` are used at the same time. I'd expect that to fail to compile.

I guess we *do* have the UI tests, and this kind of test might fit in there.